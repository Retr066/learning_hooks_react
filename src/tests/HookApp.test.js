import { shallow } from "enzyme";
import HookApp from "../HookApp";

describe("Prueba de test para el componente de HookApp", () => {
  test("El componente HookApp se tiene que renderizar correctamente", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
