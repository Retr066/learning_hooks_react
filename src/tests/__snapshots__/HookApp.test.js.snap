// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba de test para el componente de HookApp El componente HookApp se tiene que renderizar correctamente 1`] = `
<div>
  HookApp
</div>
`;
